contracts/gov/GovPool.sol

```diff
@@ -71,6 +71,9 @@ contract GovPool is
     /// @dev govPool vars
     string public descriptionURL;
 
+    mapping(uint256 => address) public proposalCreators; // proposalId => creator
+    mapping(uint256 => address) public proposalExecutors; // proposalId => executor
```

Comment:
Can be reduced to 1 storage:

`mapping(uint256 => mapping(address => uint256)) public pendingRewards;`

Then instead of "lazy" rewards, you will have to add the rewards during the `create()` `vote()` and `execute()` functions